semcore/popper/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.12.0] - 2023-11-28
+
+### Added
+
+- Opportunity to enable new focus lock scope.
```

Comment:
Let's name it "internal api" instead of "Opportunity"

Reply:
fixed